internal/db/models/task.go

```
@@ -0,0 +1,72 @@
+	}
+
+	status, err := ParseTaskStatus(str)
+	if err != nil {
+		return err
+	}
+
+	*s = status
+	return nil
+}
+
+func (s *TaskStatus) MarshalJSON() ([]byte, error) {
```

golangci-lint (1.64.8)

[warning] exported: exported method TaskStatus.MarshalJSON should have comment or be unexported

(revive)